DOC Improve web worker documentation (#798)
iodide-project/pyodide: docs/using_pyodide_from_webworker.md

`pyodide.js`, and all its associated `.asm.js`, `.data`, `.json`, and `.wasm`
files as well, though this is not strictly required if `pyodide.js` is pointing
to a site serving current versions of these files.
The simplest way to serve the required files is to use a CDN, 

> such as [pyodide-cdn2.iodide.io][cdn pyodide]. This is the solution 

Reviewer: ```suggestion
such as `https://cdn.jsdelivr.net/pyodide`. This is the solution 
```

There is probably need create a link there, as the index page will return 404 not found anyway.

presented here.

Update the `webworker.js` sample so that it has as valid URL for `pyodide.js`, and sets
`self.languagePluginUrl` to the location of the supporting files. 

In your application code create a web worker `new Worker(...)`, 
and attach listeners to it using its `.onerror` and [`.onmessage`][onmessage]
methods (listeners).

Communication from the worker to the main thread is done via the `Worker.postMessage()`
method (and vice versa).

> [cdn pyodide]: https://pyodide-cdn2.iodide.io/v0.15.0/full/

Reviewer: ```suggestion

```

In our case we will use a single worker to execute python code without interfering with 
client side rendering (which is done by the main javascript thread). The worker does 
two things:

1. Listen on new messages from the main thread 
2. Respond back once it finished executing the python script

These are the required tasks it should fulfill, but it can do other things.
For example, to always load packages `numpy` and `pytz`, you would insert the
lines `pythonLoading = self.pyodide.loadPackage(['numpy', 'pytz'])` and 
`await pythonLoading;` as shown below:

```js 
// webworker.js 

// Setup your project to serve `py-worker.js`. You should also serve
// `pyodide.js`, and all its associated `.asm.js`, `.data`, `.json`, 
// and `.wasm` files as well:

> self.languagePluginUrl = 'https://pyodide-cdn2.iodide.io/v0.15.0/full/';
> importScripts('https://pyodide-cdn2.iodide.io/v0.15.0/full/pyodide.js');

Reviewer: ```suggestion
self.languagePluginUrl = 'https://cdn.jsdelivr.net/pyodide/v0.15.0/full/';
importScripts('https://cdn.jsdelivr.net/pyodide/v0.15.0/full/pyodide.js');
```
